src/editor/SegmentationSettings.js

```diff
@@ -0,0 +1,50 @@
+/**
```

Comment:
Noted elsewhere, I think all of the Minimum Number of Posts UI should be removed for now.

Reply:
Done in 4c9b9149a365c44db8c156af6672fd7af2ce69d4